x/sequencer/keeper/msg_server_increase_bond.go

```diff
@@ -0,0 +1,54 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+)
+
+// IncreaseBond implements types.MsgServer.
+func (k msgServer) IncreaseBond(goCtx context.Context, msg *types.MsgIncreaseBond) (*types.MsgIncreaseBondResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// check if the sequencer already exists
+	sequencer, found := k.GetSequencer(ctx, msg.Creator)
+	if !found {
+		return nil, types.ErrUnknownSequencer
+	}
+
+	// check if the sequencer is bonded
+	if !sequencer.IsBonded() {
+		return nil, types.ErrInvalidSequencerStatus
+	}
+
+	// check if sequencer is currently jailed
+	if sequencer.Jailed {
+		return nil, types.ErrSequencerJailed
+	}
```

Comment:
this block could probably go into its own method, as it could probably be reused